src/status_im/ui/screens/browser/views.cljs

```diff
@@ -100,7 +100,9 @@
                                 :handler   #(re-frame/dispatch [:navigate-to-modal :wallet-modal])}]]]
        [react/view components.styles/flex
         [components.webview-bridge/webview-bridge
-         {:ref                                   #(reset! webview %)
+         {:dapp?                                 dapp?
```

Comment:
do we keep state only for dapps from our list?

Reply:
right now only for dapps, but we can use any other id instead of dapp-name